java-frontend/src/test/java/org/sonar/java/se/JavaCheckVerifier.java

```diff
@@ -177,16 +177,20 @@ public static void verifyIssueOnFile(String filename, String message, JavaFileSc
     javaCheckVerifier.scanFile(filename, new JavaFileScanner[] {check});
   }
 
-  public void scanFile(String filename, JavaFileScanner[] checks) {
+  public static Collection<File> classpathFromDirectory(String jarsDirectory) {
```

Comment:
there's no need to change this class : your test should include only the required dependency.